sales_team_security_sale/security/security.xml

```diff
@@ -1,42 +1,39 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<odoo>
-    <data noupdate="1">
-
+<odoo noupdate="1">
```

Comment:
If you do this, the indentation is now incorrect and provokes the rest of the changes. Is that avoidable?

Reply:
Don't join both tags as stated in the other twin PR